lms/djangoapps/courseware/views.py

```diff
@@ -395,6 +395,9 @@ def _index_bulk_op(request, course_key, chapter, section, position):
             'reverifications': fetch_reverify_banner_info(request, course_key),
         }
 
+        # Check for course required content (such as an Entrance Exam)
+        chapter, section = override_with_required_content(course_module, course, request.user, chapter, section)
```

Comment:
Also, I'm wondering if this might be more cleanly implemented by a a simple redirect rather than aliasing content at some other's url. It seems like a bit of a violation of HTTP standards to alias content under a different URL. Not sure of any client-side caching considerations by this approach.